network/anchor-platform/admin-guide/sep6/configuration.mdx

```diff
@@ -101,10 +209,18 @@ SEP1_ENABLED=true
 SEP1_TOML_TYPE=file
 SEP1_TOML_VALUE=/home/dev.stellar.toml
 
+SEP6_ENABLED=true
+SEP6_MORE_INFO_URL_BASE_URL=http://example.com
+SECRET_SEP6_MORE_INFO_URL_JWT_SECRET="your encryption key shared with your business server"
```

Comment:
I'm not sure if we want to add these environment variables to the guide, since the more info page is an optional feature. @Ifropc what do you think?

Reply:
FWIW it's also present in the SEP-24 configuration docs. We don't have any references to the `_MORE_INFO` configs anywhere besides in the AP repo, so we should call them out somewhere if not here.

Reply:
I think we should add comment for optional features IMO